ReactiveCocoaTests/Swift/PropertySpec.swift

```diff
@@ -14,66 +14,258 @@ import ReactiveCocoa
+
+				var sentValue: String?
+				var signalCompleted = false
+
+				constantProperty.producer.start(next: { value in
+					sentValue = value
+				},
+				completed: {
+					signalCompleted = true
+				})
+
+				expect(sentValue).to(equal(propertyValue))
+				expect(signalCompleted).to(beTruthy())
 			}
 		}
 
 		describe("MutableProperty") {
-			pending("should have the value given at initialization") {
+			it("should have the value given at initialization") {
+				let propertyValue = "StringValue"
+				let mutableProperty = MutableProperty(propertyValue)
+
+				expect(mutableProperty.value).to(equal(propertyValue))
 			}
 
-			pending("should yield a producer that sends the current value then all changes") {
+			it("should yield a producer that sends the current value then all changes") {
+				let initialValue = "StringValue"
+				let subsequentValue = "NewStringValue"
+
+				let mutableProperty = MutableProperty(initialValue)
+
+				var sentValue: String?
+				var signalCompleted = false
+
+				mutableProperty.producer.start(next: { value in
+					sentValue = value
+				},
+				completed: {
+					signalCompleted = true
+				})
+
+				expect(sentValue).to(equal(initialValue))
+				expect(signalCompleted).to(beFalsy())
+
+				mutableProperty.value = subsequentValue
+
+				expect(sentValue).to(equal(subsequentValue))
+				expect(signalCompleted).to(beFalsy())
 			}
 
-			pending("should complete its producer when deallocated") {
+			it("should complete its producer when deallocated") {
+				let propertyValue = "StringValue"
+				var mutableProperty: MutableProperty? = MutableProperty(propertyValue)
+
+				var signalCompleted = false
+
+				mutableProperty?.producer.start(next: { value in
+				},
+				completed: {
+					signalCompleted = true
+				})
+
+				mutableProperty = nil
+				expect(signalCompleted).to(beTruthy())
 			}
 		}
 
 		describe("PropertyOf") {
-			pending("should pass through behaviors of the input property") {
+			it("should pass through behaviors of the input property") {
+				let propertyValue = "StringValue"
+
+				let constantProperty = ConstantProperty(propertyValue)
+				let propertyOf = PropertyOf(constantProperty)
+
+				var sentValue: String?
+
+				propertyOf.producer.start(next: { value in
+					sentValue = value
+				})
+
+				expect(sentValue).to(equal(propertyValue))
 			}
 		}
 
 		describe("binding") {
 			describe("from a Signal") {
-				pending("should update the property with values sent from the signal") {
+				it("should update the property with values sent from the signal") {
+					let finalPropertyValue = 815
+					var signalObserver: SinkOf<Event<Int, NoError>>?
+
+					let sendPropertyUpdate = {
+						sendNext(signalObserver!, finalPropertyValue)
+					}
+
+					let signal = Signal<Int, NoError>({ observer in
+						signalObserver = observer
+
+						return SimpleDisposable()
+					})
+
+					let mutableProperty = MutableProperty<Int>(0)
+
+					mutableProperty <~ signal
```

Comment:
We should probably verify that the property has the same value immediately after this binding.